transports/graphql.go

```diff
@@ -865,6 +865,43 @@ func (g *TransportGraphQL) UpdateTransactionMetadata(ctx context.Context, txID s
 	return respData.Transaction, nil
 }
 
+// GetUtxo will get a utxo by transaction ID
+func (g *TransportGraphQL) GetUtxo(ctx context.Context, txID string, outputIndex uint32) (*buxmodels.AccessKey, ResponseError) {
```

Comment:
Isn't this supposed to return `buxmodels.Utxo` instead of AccessKey? I'm not sure, but the tests are not passing.

Reply:
@kuba-4chain  Yes, this should have been  `buxmodels.Utxo` 🤦